packages/neon_framework/packages/account_repository/lib/src/account_storage.dart

```diff
@@ -3,52 +3,205 @@ part of 'account_repository.dart';
 /// {@template account_repository_storage}
 /// Storage for the [AccountRepository].
 /// {@endtemplate}
-@immutable
-class AccountStorage {
+class AccountStorage with Table {
   /// {@macro account_repository_storage}
-  const AccountStorage({
-    required this.accountsPersistence,
-    required this.lastAccountPersistence,
-  });
+  AccountStorage();
 
-  /// The store for the account list.
-  final SingleValueStore accountsPersistence;
+  @override
+  String get name => 'account_credentials';
 
-  /// The store for the last used account.
-  final SingleValueStore lastAccountPersistence;
+  @override
+  int get version => 1;
+
+  /// Column for the [Credentials.serverURL].
+  @protected
+  static const String serverURL = 'server_urlL';
+
+  /// Column for the [Credentials.username].
+  @protected
+  static const String loginName = 'login_name';
+
+  /// Column for the [Credentials.appPassword].
+  @protected
+  static const String appPassword = 'app_password';
+
+  /// Column to store the active account.
+  @protected
+  static const String active = 'active';
+
+  @override
+  void onCreate(Batch db, int version) {
+    db.execute('''
+CREATE TABLE "$name" (
+  "$serverURL"      TEXT NOT NULL,
+  "$loginName"      TEXT NOT NULL,
+  "$appPassword"    TEXT,
+  "$active"         BOOLEAN NOT NULL DEFAULT 0,
```

Comment:
We should store this separately as it is not really an information about the account itself.
Also there is no check that two accounts can not be active at the same time.